packages/frontend/src/utils/zk-catalog/trustedSetup.ts

```diff
@@ -0,0 +1,20 @@
+import { ZkCatalogOnchainVerifier } from './types'
+
+export function trustedSetup(verifiers: ZkCatalogOnchainVerifier[]) {
```

Comment:
well, this could be named `getTrustedSetup` or smth. We do not have this kind of pattern in our codebase